smf/ntp/manifest/manifest.xml

```diff
@@ -57,29 +62,20 @@
     The service also always starts the binary with ASLR enabled,
     regardless of whether it was linked with -zaslr
   -->
-  <exec_method type="method" name="start" exec="/var/svc/method/svc-site-ntp %m"
-    timeout_seconds="60">
+
+  <exec_method type="method" name="start" 
+    exec='/usr/sbin/chronyd -d &amp;' 
+    timeout_seconds="600">
```

Comment:
NB: Because you're using the ampersand, the timeout is almost irrelevant; the shell will put it in the background and exit immediately, whether it started correctly or not.

Reply:
ha! good to know